--- jemalloc-3.0.0/Makefile.in.orig	2012-05-12 02:50:38.000000000 +0200
+++ jemalloc-3.0.0/Makefile.in	2012-05-23 10:15:00.123902010 +0200
@@ -69,7 +69,7 @@
 LIBJEMALLOC := $(LIBPREFIX)jemalloc$(install_suffix)
 
 # Lists of files.
-BINS := $(srcroot)bin/pprof $(objroot)bin/jemalloc.sh
+BINS := $(objroot)bin/jemalloc.sh
 CHDRS := $(objroot)include/jemalloc/jemalloc$(install_suffix).h \
 	$(objroot)include/jemalloc/jemalloc_defs$(install_suffix).h
 CSRCS := $(srcroot)src/jemalloc.c $(srcroot)src/arena.c $(srcroot)src/atomic.c \
