--- jemalloc-3.5.0/Makefile.in.orig	2014-01-27 16:56:57.474782211 +0100
+++ jemalloc-3.5.0/Makefile.in	2014-01-27 16:58:18.348112118 +0100
@@ -73,7 +73,7 @@
 LIBJEMALLOC := $(LIBPREFIX)jemalloc$(install_suffix)
 
 # Lists of files.
-BINS := $(srcroot)bin/pprof $(objroot)bin/jemalloc.sh
+BINS := $(objroot)bin/jemalloc.sh
 C_HDRS := $(objroot)include/jemalloc/jemalloc$(install_suffix).h
 C_SRCS := $(srcroot)src/jemalloc.c $(srcroot)src/arena.c \
 	$(srcroot)src/atomic.c $(srcroot)src/base.c $(srcroot)src/bitmap.c \
